packages/file.vm/tools/chocolateyinstall.ps1

```diff
@@ -8,8 +8,10 @@ try {
     $zipUrl = "https://github.com/nscaife/file-windows/releases/download/20170108/file-windows-20170108.zip"
     $zipSha256 = "963147318f96d9345471e1a9a3943def4d95fcb3c1fe020e465ab910d0cda4a3"
 
-    $executablePath = (VM-Install-From-Zip $toolName $category $zipUrl -zipSha256 $zipSha256 -consoleApp $true)[-1]
-    $executableDir = Split-Path -Path $executablePath
+    VM-Install-From-Zip $toolName $category $zipUrl -zipSha256 $zipSha256 -consoleApp $true
```

Comment:
Same here. We should leave it as it was before and fix `VM-Install-From-Zip` to behave as expected.